src/screens/CreatePost/elements/PollItem.js

```diff
@@ -19,7 +19,7 @@ export default function PollItem({
   return (
     <View style={isTextInputFocus ? S.focuspollitemcontainer : S.pollitemcontainer}>
       <TextInput
-        placeholderTextColor={COLORS.blackgrey}
+        placeholderTextColor={COLORS.gray400}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.blackgrey` to `COLORS.gray400` for the `placeholderTextColor` prop in the `TextInput` component seems fine. However, ensure that `COLORS.gray400` is defined and provides a better contrast ratio for accessibility purposes.
```diff
- placeholderTextColor={COLORS.blackgrey}
+ placeholderTextColor={COLORS.gray400}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->